tasks/configure-Debian.yml

```diff
@@ -38,6 +38,17 @@
   notify: restart apache
   when: apache_create_vhosts
 
+- name: Configure Apache (Add listen port for every vhost).
+  blockinfile:
+    dest: "{{ apache_server_root }}/conf/{{ apache_daemon }}.conf"
+    insertafter: "^Listen {{ apache_listen_port }}"
+    block: "{% set return = joiner(\"\\n\") %}{% for item in apache_vhosts %}{% if item.listenport is defined and apache_listen_port != item.listenport %}{{ return() }}Listen {{ item.listenport }}{% endif %}{% endfor %}"
```

Comment:
I agree the "block" is not very simple to read.
I make this quickly, but there is perhaps a better way to do this.
I have try with with_items which is more simple to read but it's overide the line, so you have only the last entry.
Have you an idea for somethings more easy ? I could try it